### DISCUSSION TITLE
Add-on support

### DISCUSSION THREAD

**Discussion by HexBugOrion** [terrarium-earth/Ad-Astra#19] [Ideas]
I'm not sure if it's already here but making addons and whatnot could be cool! I was asking because I wanted to add origins hooks for beings who can survive without oxygen or have different powers that work with the moon's day-night cycle, and so on. Plus adding more variety to building materials and airlocks sounds neat.

**Comment by AlexNijjar** [comment]
There’s no support yet, but I want to put this on a maven eventually for mod devs to make addon mods.

**Comment by AlexNijjar** [comment]
it's on the maven now. check the readme and wiki

**Comment by CodexAdrian** [comment]
Ad Astra and its dependencies are all hosted on the Resourceful Bees maven
